### ISSUE TITLE
IBAN in UK does not function locally

### ISSUE THREAD

**Issue by ser** [bisq-network/bisq#405]
Hi, 
It seems that national bank setup in the UK is really confusing, and in my opinion it's pretty serious confusion which is able to prevent most of trades. In the UK we usually do not use IBAN, to transfer money we need only two functions: "account number" and "sort code". But Bitsquare requires IBAN, look at the screenshot:
![selection_015](https://cloud.githubusercontent.com/assets/231537/14838229/667a26f2-0c49-11e6-9433-8802897eda15.png)
And, if I put IBAN, instead of just account number, most of traders would not understand that "99228833" is my account number, which is required for a national bank transfer. Most of banks do not accept IBAN format.

In summary: it's required to remove phrase "IBAN" from UK bank account setup.

**Comment by ManfredKarrer** [comment]
"The International Bank Account Number (IBAN) is an internationally agreed system of identifying bank accounts across national borders to facilitate the communication and processing of cross border transactions with a reduced risk of transcription errors."
https://en.wikipedia.org/wiki/International_Bank_Account_Number

So far the theory, but banks love the 70s...
I love the 70s as well but more the music not the tech :-)

Yes I am aware that the form is not optimal for each country. It is unfortunately quite a bit of work and I cannot squeeze that in soon. Need some additional devs to help out with all the UI stuff...

Will change the label to Account NR. or IBAN for now ....

**Comment by AndreKoster** [comment]
@ser At first, I was a bit surprised to hear that the UK would be lagging behind so much. However, a google search showed that major UK banks like HSBC, Barclays, RBS, all have elaborate information about IBAN on their websites, and claim that someone's IBAN account number is present on all bank statements.

Hence, in individual cases it's just a matter of people educating themselves. The information is easy to find.

Sources:
http://www.business.hsbc.uk/en-gb/payments/clearing-and-foreign-currency-payments/iban-and-bic
http://www.rbs.co.uk/corporate/international/g0/guide-to-international-business/regulatory-information/iban.ashx
https://www.help.barclays.co.uk/faq/international/payments/making-and-receiving-international-payments/what-iban-swiftcode.html

**Comment by ser** [comment]
Hi,
to be clear: it is _impossible_ to make a national bank transfer in UK using IBAN. 

It's screen shot from one of online banks, if you put IBAN as account number, money will not reach the recipient. Account number is exactly eight numbers only.

![selection_017](https://cloud.githubusercontent.com/assets/231537/14896083/379b4dee-0da6-11e6-81dd-a667ae7e4c72.png)

Transforming IBAN to sort code plus account number not always is easy, and here in the UK no person uses IBAN nationally. Eventually it is helpful when you send money abroad, but only if you use SEPA, not other means of banking.

**Comment by ManfredKarrer** [comment]
I changed the wording, so you can use the field for account nr. or IBAN.
Will close that issue and as soon I have more support on UI dev side the payment forms will be adjusted to each country. Will close for now the issue.
